modules/serverless-ossm-secret-filtering-net-kourier.adoc

```diff
@@ -21,10 +23,6 @@ If you enable secret filtering, all of your secrets need to be labeled with  `ne
 * Install the {ServerlessOperatorName} and Knative Serving.
 * Install the OpenShift CLI (`oc`).
 
-.Procedure
-
-* Set the `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID` variable to `true` for `net-kourier-controller` in the `KnativeServing` CR:
-+
 .Example KnativeServing CR
```

Comment:
I think this example should be removed. Not just the "procedure" above. The example was there to show how to enable the filtering. We only need the new example below. Both together don't make sense.

Reply:
Otherwise these docs look good to me now.

Reply:
[mgencur](https://github.com/mgencur) Does that mean I need to remove the example from the `net-istio` section as well, right?

Reply:
@kaldesai TBH. I'm not sure. The example for net-istio has some other parts that are important. But maybe that could be merged together. Right now we have this:
![image](https://github.com/openshift/openshift-docs/assets/336823/ba0e42f7-a01a-4865-92a2-4cdb345e8532)
But we could have just one example:
```
apiVersion: operator.knative.dev/v1beta1
kind: KnativeServing
metadata:
  name: knative-serving
  namespace: knative-serving
spec:
  ingress:
    istio:
      enabled: true
  workloads:
    - annotations:
        sidecar.istio.io/inject: "true"
        sidecar.istio.io/rewriteAppHTTPProbers: "true"
      name: activator
    - annotations:
        sidecar.istio.io/inject: "true"
        sidecar.istio.io/rewriteAppHTTPProbers: "true"
      name: autoscaler
    - env:
     - container: controller
       envVars:
       - name: ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID
         value: 'false'
     name: net-istio-controller
```
I think both ways are ok-ish.

Reply:
Okay, so I will use the merged one. The one which you provided here.